internal/host_endianess_le.go

```diff
@@ -0,0 +1,7 @@
+//go:build !(armbe || arm64be || mips || mips64 || mips64p32 || ppc || ppc64 || s390 || s390x || sparc || sparc64)
```

Comment:
I'm not sure if this is how it works in Go (and I can't test atm), but maybe it would be better to explicitly have the LE architectures here, so if a new one gets introduced and telegraf tries to build for it, the compilation will (hopefully) fail instead of falling back to LE as a default (which might be incorrect).

Reply:
This works, but having an explicit list might also be helpful. I'm fine either way, @powersj what do you think?

Reply:
Bad wording on my part: by "that works" I meant whether the compilation will work for specified architectures and fail for unspecified ones.

Reply:
See (applied) suggestion of @powersj...